canonical-web-and-design/ubuntu.com
# /blog endpoints with Flask

Make use of a new version of canonicalwebteam.blog to set up all the blog endpoints.

This branch currently points to [a new branch on canonicalwebteam.blog](https://github.com/nottrobin/canonicalwebteam.blog/tree/expose-blogviews). QAing this PR can be a QA for [this PR for v0.3.0](https://github.com/canonical-web-and-design/canonicalwebteam.blog/pull/118).

Fixes https://github.com/canonical-web-and-design/base-squad/issues/689

QA
--

Go to `/blog`, `/blog/cloud-and-server`, `/blog/topics/design` etc. etc., check everything seems to work fine.
